deepchecks/vision/metrics_utils/scorers.py

```diff
@@ -140,9 +140,15 @@ def get_scorers_dict(
                     raise DeepchecksNotSupportedError(
                         f'Unsupported metric: {name} of type {type(metric).__name__} was given.')
                 scorers[name] = converted_met
+            elif isinstance(metric, t.Callable):
+                if task_type == TaskType.CLASSIFICATION:
+                    scorers[name] = CustomClassificationScorer(metric)
+                else:
+                    raise DeepchecksNotSupportedError('Custom scikit-learn scorers are only supported for'
```

Comment:
its not just scikit-learn scorers, all callables only work in classification

Reply:
we don't want them passing any callable that is not an sklearn scorer. Can't enforce that though.

Reply:
IMO you want all callable in sklearn format (not necessarily just stuff imported fro sklearn)
Can maybe just check that the callable expects the relevant arguments? not that important